source/_components/lock.template.markdown

```diff
@@ -36,7 +37,12 @@ lock:
       service: switch.turn_off
       data:
         entity_id: switch.door
+    availability_template: >-
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.